core/src/main/scala/org/apache/spark/storage/BlockManagerMasterEndpoint.scala

```diff
@@ -168,6 +177,37 @@ class BlockManagerMasterEndpoint(
       stop()
   }
 
+  private def handleFailure[T](
```

Comment:
I think this function could use a docstring.

Reply:
What's `docstring`? Could you explain more?

Reply:
javadoc :-)

Reply:
I see, thanks.